src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/EditingPreferencesPane.java

```diff
@@ -535,6 +590,8 @@ else if (isSublime)
       prefs_.foldStyle().setGlobalValue(foldMode_.getValue());
       prefs_.surroundSelection().setGlobalValue(delimiterSurroundWidget_.getValue());
       prefs_.executionBehavior().setGlobalValue(executionBehavior_.getValue());
+      prefs_.autoSaveOnIdle().setGlobalValue(autoSaveOnIdle_.getValue());
+      prefs_.autoSaveIdleMs().setGlobalValue(Integer.parseInt(autoSaveIdleMs_.getValue()));
```

Comment:
Do you think it's worth using `StringUtil.parseInt(<...>, 1000)` here, just to avoid a potential exception from a parse failure? (It looks like invalid options are handled in the dialog directly but being paranoid doesn't hurt here)

Reply:
Good idea!